Automattic/jetpack
# SSO: Log error message from user invite response

## Proposed changes:

* Adds error message to event tracking if error occurs when sending user invite.
* Displays API error message on UI for user invite.
* Copy changes to Connection packages where SSO code changes will be located eventually.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync changes to Atomic site
* Add a new user if you haven't already, navigate to `/wp-admin/users.php`, then send an invite. Verify that its working correctly.
* FTP into your atomic site and add the following to your `wp-config.php` file: `define('JETPACK__SANDBOX_DOMAIN', 'SANDBOX_DOMAIN_HERE');`
* Force the invite_user endpoint to return an error and verify the error message is tracked correctly at mission control `/tracks/live` under the `sso_user_invite_sent` event.
